src/main/proto/netflix/titus/titus_eviction_api.proto

```diff
@@ -0,0 +1,135 @@
+syntax = "proto3";
+
+// WORK IN PROGRESS
+// This API is likely to change.
+
+// Titus task eviction API specification.
+//
+
+package com.netflix.titus;
+
+import "google/protobuf/empty.proto";
+import "netflix/titus/titus_base.proto";
+
+option java_multiple_files = true;
+option java_package = "com.netflix.titus.grpc.protogen";
+option java_outer_classname = "EvictionProto";
+
+option go_package = "titus";
+
+// ----------------------------------------------------------------------------
+// Eviction core data structures
+
+/// The system distruption budget provides an upper bound on allowed number of task evictions. The system
+//  disruption budget is defined at the global, tier and a capacity group level.
+message SystemDisruptionBudget {
+    /// System layer/entity refrence.
+    Reference reference = 1;
+
+    /// (Required) The admision control policy.
+    TokenBucketPolicy admissionControlPolicy = 2;
+}
+
+/// Current eviction quota at the global, tier, capacity group, and job level.
+//  For example, for the global disruption budget with the bucket size of 100 half empty, the eviction quota is 50.
+message EvictionQuota {
+    /// System layer/entity refrence.
+    Reference reference = 1;
+
+    /// (Required) Total number of tasks that can be evicted at a given point in time.
+    uint32 quota = 2;
+}
+
+// ----------------------------------------------------------------------------
+// GRPC services
+
+message CapacityGroupId {
+    string id = 1;
+}
+
+message TierId {
+    Tier tier = 1;
+}
+
+message TaskTerminateRequest {
```

Comment:
should this be moved to `titus_base.proto`, so we can reuse it on the `JobManagementService` as well (`KillTask`)?

Reply:
The particular use case is when tasks are being terminated by a monitoring system. It would be good to capture that information so we can act on it later (e.g.: by quarantining and evicting bad agents with too many tasks being killed by a monitoring system).

Reply:
I do not think we should do it, as this is a wrapper type for `EvictionService`.